dask/array/core.py

```diff
@@ -694,6 +694,8 @@ def map_blocks(
         dtype = apply_infer_dtype(func, args, original_kwargs, "map_blocks")
 
     if drop_axis:
+        if any(i < 0 for i in drop_axis): 
+            raise ValueError("Drop_axis is {0}, expected axis to be non-negative.".format(drop_axis))
```

Comment:
Nitpick, but I find f strings easier to read.

```suggestion
            raise ValueError(f"Drop_axis is {drop_axis}, expected axis to be non-negative.")
```

Reply:
Thanks @jsignell - I'll add these changes in. Sorry for the late reply btw!